public/components/game/details/details.stache

```diff
@@ -30,8 +30,13 @@
 			<td>{{name}}</div>
 			<td class="stats-container">
 				{{#eachOf statsForPlayerId(id)}}
-					<span ($click)="gotoTimeMinus5( time, %event)" 
-							class='stat-point stat-{{type}}' style="left: {{statPercent time}}%">{{type}}</span>
+					<span ($click)="gotoTimeMinus5(time, %event)"
+						class="stat-point stat-{{type}}" style="left: {{statPercent time}}%">
+						{{type}}
+						{{#if session.isAdmin}}
+							<span class="destroy-btn glyphicon glyphicon-trash" ($click)="deleteStat"></span>
```

Comment:
It needs to stop propagation also.

On Friday, March 4, 2016, Jan Jorgensen <notifications@github.com> wrote:

> In public/components/game/details/details.stache
> <https://github.com/donejs/bitballs/pull/106#discussion_r55083760>:
>
> > @@ -30,8 +30,13 @@
> >  			<td>{{name}}</div>
> >  			<td class="stats-container">
> >  				{{#eachOf statsForPlayerId(id)}}
> > -					<span ($click)="gotoTimeMinus5( time, %event)"
> > -							class='stat-point stat-{{type}}' style="left: {{statPercent time}}%">{{type}}</span>
> > +					<span ($click)="gotoTimeMinus5(time, %event)"
> > +						class="stat-point stat-{{type}}" style="left: {{statPercent time}}%">
> > +						{{type}}
> > +						{{#if session.isAdmin}}
> > +							<span class="destroy-btn glyphicon glyphicon-trash" ($click)="deleteStat"></span>
>
> Any reason it can't just be ($click)="destroy()?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/donejs/bitballs/pull/106/files#r55083760>.
>